examples/webextension/background.js

```diff
@@ -0,0 +1,15 @@
+const TRACKING_ID = 'UA-XXXXXXXX-YY';
+
+const { sendEvent } = new Metrics({
```

Comment:
Yeah, AFAIK, there is no module system built into WebExtensions. We're using rollup & babel on Snooze Tabs - but it *is* good not to assume that will be the case.